force-app/main/default/aura/RD2_EnablementDelegate/RD2_EnablementDelegateController.js

```diff
@@ -28,9 +28,9 @@
         helper.handleBatchEvent(component, event, 'v.dryRunBatch');
         helper.refreshDryRun(component);
         helper.refreshEnable(component);
-        var status = event.Hp.batchProgress.status;
+        var status = (event.Hp && event.Hp.batchProgress) ? event.Hp.batchProgress.status : undefined;
```

Comment:
I was going to suggest simplifying this to `var status = event?.Hp?.batchProgress?.status;` but apparently that doesn't work in aura 😞

Reply:
Yeah. I found the same